CONTRIBUTING.md

```diff
@@ -26,10 +26,22 @@ Clone the repository, enter the directory, and build the provider:
 ```sh
 $ git clone git@github.com:hashicorp/terraform-provider-tfe
 $ cd terraform-provider-tfe
-$ make build
+$ make
 ```
 
-This will build the provider and put the binary in the `$GOPATH/bin` directory. To use the compiled binary, you have several different options (this list is not exhaustive):
+This will build the provider and put the binary in the project directory. To use the compiled binary, you have several different options (this list is not exhaustive):
+
+##### Using CLI config to provide a dev override (Using Terraform v0.14+)
+
+Use the rule `make devoverride` to generate a CLI config containing a dev override provider installation. This command will print a variable export that can be copied and pasted into a shell session while testing with terraform. To automatically export this override, use `eval $(make devoverride)`
+
+Example usage:
+
+```sh
+$ eval $(make devoverride)
+$ cd ../example-config
```

Comment:
💅 I think it might be helpful to update the name of the directory in the example:
```suggestion
$ cd ../example-terraform-config
```